test/test_average_pooling_layer.h

```diff
@@ -1,111 +1,73 @@
-    notice, this list of conditions and the following disclaimer.
-    * Redistributions in binary form must reproduce the above copyright
-    notice, this list of conditions and the following disclaimer in the
-    documentation and/or other materials provided with the distribution.
-    * Neither the name of the <organization> nor the
-    names of its contributors may be used to endorse or promote products
-    derived from this software without specific prior written permission.
-
-    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
-    EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY
-    DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
-    (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
-    ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-*/
+// Copyright (c) 2013-2016, Taiga Nomi. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 #pragma once
 #include "picotest/picotest.h"
 #include "testhelper.h"
 #include "tiny_dnn/tiny_dnn.h"
 
 namespace tiny_dnn {
 
-TEST(ave_pool, gradient_check) { // sigmoid - cross-entropy
-    typedef cross_entropy loss_func;
-    typedef activation::sigmoid activation;
-    typedef network<sequential> network;
+TEST(ave_pool, gradient_check) {  // sigmoid - cross-entropy
+  typedef cross_entropy loss_func;
+  typedef activation::sigmoid activation;
+  typedef network<sequential> network;
 
-    network nn;
-    nn << fully_connected_layer<activation>(3, 8)
-        << average_pooling_layer<activation>(4, 2, 1, 2); // 4x2 => 2x1
+  network nn;
+  nn << fully_connected_layer<activation>(3, 8)
+     << average_pooling_layer<activation>(4, 2, 1, 2);  // 4x2 => 2x1
 
-    const auto test_data = generate_gradient_check_data(nn.in_data_size());
-    nn.init_weight();
+  const auto test_data = generate_gradient_check_data(nn.in_data_size());
+  nn.init_weight();
 
-    EXPECT_TRUE(nn.gradient_check<loss_func>(test_data.first, test_data.second, epsilon<float_t>(), GRAD_CHECK_ALL));
+  EXPECT_TRUE(nn.gradient_check<loss_func>(test_data.first, test_data.second,
+                                           epsilon<float_t>(), GRAD_CHECK_ALL));
 }
 
 TEST(ave_pool, forward) {
-    average_pooling_layer<identity> l(4, 4, 1, 2);
-    vec_t in = {
-        0, 1, 2, 3,
-        8, 7, 5, 6,
-        4, 3, 1, 2,
-        0,-1,-2,-3
-    };
-
-    vec_t expected = {
-        4, 4,
-        1.5, -0.5
-    };
```

Comment:
Writing clang-format option comments in source code sounds a bit weird.
Do we have clear formatting rules before applying `clang-format`?